cli/cmd/start.go

```diff
@@ -105,8 +99,8 @@ func internalStartModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 
 	checkPeriod := time.Duration(0)
 
-	if cmdCtx.Cli.IntegrityCheck != "" && integrityCheckPeriod > 0 {
-		checkPeriod = time.Duration(integrityCheckPeriod * int(time.Second))
+	if cmdCtx.Cli.IntegrityCheck != "" && cmdCtx.Cli.IntegrityCheckPeriod > 0 {
+		checkPeriod = time.Duration(cmdCtx.Cli.IntegrityCheckPeriod * int(time.Second))
 	}
 
 	if err := running.Start(cmdCtx, &runningCtx.Instances[0], checkPeriod); err != nil {
```

Comment:
At now for each commend we will need to implement the global flag pass into `Start()`. It's better to use the global flag here:

https://github.com/tarantool/tt/blob/7a920f35d986536ddde6c45f62391d97b95579f6/cli/running/running.go#L721-L723

directly. It will allow us not to change a code of `restart` as example.

Reply:
Do you mean to use global flag directly passing context into `running.Start()` and use `cmdCtx.Cli.IntegrityCheckPeriod` directly in this function instead of `integrityCheckPeriod` variable?

Reply:
Yes, I understood, fixed it.